source/_integrations/alarm_control_panel.markdown

```diff
@@ -10,3 +10,31 @@ ha_domain: alarm_control_panel
 
 Home Assistant can give you an interface which is similar to a classic alarm system.
 Please see [manual alarm](/integrations/manual) for alarm configuration.
+
+### Attributes
+
+Alarm system exposes the following attributes:
+
+- `code_format` - i.e., `text`, `number` or `null` if no code is needed
```

Comment:
This is a frontend concern, why would we list that here?

Reply:
Mostly for the sake of completeness. Will remove...